lib/open_project/translations/models/git_repository.rb

```diff
@@ -14,7 +14,17 @@ def clone
 
   def checkout(ref)
     within_repo do
-      run_command "git checkout --force '#{ref}' --"
+      begin
+        run_command "git checkout --force '#{ref}' --"
+      rescue Exception => e
```

Comment:
Avoid rescuing the `Exception` class. Perhaps you meant to rescue `StandardError`?